dev-python/sphinxcontrib-svg2pdfconverter/sphinxcontrib-svg2pdfconverter-1.2.3.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit distutils-r1
+
+DESCRIPTION="Sphinx SVG to PDF converter extension"
+HOMEPAGE="https://github.com/missinglinkelectronics/sphinxcontrib-svg2pdfconverter"
+SRC_URI="
+	https://github.com/missinglinkelectronics/sphinxcontrib-svg2pdfconverter/archive/v${PV}.tar.gz
+		-> ${P}.gh.tar.gz
+"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="cairosvg"
```

Comment:
what does this flag do? it is not used in the ebuild.

Reply:
Used in RDEPEND, but maybe there is better way. This package needs only one of three backends at runtime: inkscape, librsvg, cairosvg.

Reply:
That is not a valid case for using a USE flag, this can be done with and `|| ( ... )` dependency.